moveit_planners/ompl/ompl_interface/src/model_based_planning_context.cpp

```diff
@@ -1000,8 +1000,17 @@ int32_t ompl_interface::ModelBasedPlanningContext::logPlannerStatus(const og::Si
       result = moveit_msgs::msg::MoveItErrorCodes::TIMED_OUT;
       break;
     case ompl::base::PlannerStatus::APPROXIMATE_SOLUTION:
-      RCLCPP_WARN(LOGGER, "Solution is approximate");
-      result = moveit_msgs::msg::MoveItErrorCodes::SUCCESS;
+      // timeout is a common reason for APPROXIMATE_SOLUTION
+      if (ompl_simple_setup->getLastPlanComputationTime() > request_.allowed_planning_time)
+      {
+        RCLCPP_WARN(LOGGER, "Planning timed out");
```

Comment:
Nit: Can we print the planning timeout value here along with the `Planning timed out` message?
```suggestion
        RCLCPP_WARN(LOGGER, "Planning timed out. Allowed planning time is currently set to %f", request_.allowed_planning_time
```